x/mongo/driver/operation.go

```diff
@@ -387,9 +387,11 @@ func (op Operation) Execute(ctx context.Context, scratch []byte) error {
 			connDesc := conn.Description()
 			retryableErr := tt.Retryable(connDesc.WireVersion)
 			preRetryWriteLabelVersion := connDesc.WireVersion != nil && connDesc.WireVersion.Max < 9
-
-			// If retry is enabled, add a RetryableWriteError label for retryable errors from pre-4.4 servers
-			if retryableErr && preRetryWriteLabelVersion && retryEnabled {
+			inTransaction := !(op.Client.Committing || op.Client.Aborting) &&
+				(op.Client.TransactionInProgress() || op.Client.TransactionStarting())
```

Comment:
Can `(op.Client.TransactionInProgress() || op.Client.TransactionStarting())` be changed to [`op.Client.TransactionRunning()`](https://github.com/mongodb/mongo-go-driver/blob/v1.4.0-beta2/x/mongo/driver/session/client_session.go#L256)? Same for the case below.